hs/web_parsing/ArtSciParser.hs

```diff
@@ -82,63 +88,29 @@ OUTPUT: Course 'record' containing course info
 ----------------------------------------------------------------------------------------}
 processCourseToData :: [Tag String] -> Course
 processCourseToData tags  =
-    let cleanTags = map cleanTag tags
-        getTitle = find (~== TagText "") cleanTags
-        --splitat 8 since first 8 represent course code
-        courseNames = splitAt 8 $ removeTitleGarbage $ removeLectureSection $ getTitle
-    in Course {breadth = Nothing, 
+    let course = 
+          Course {breadth = Nothing, 
             description = Nothing, 
-            --we drop 1 to remove space between title and course 
-            title  = (Just (T.pack $ drop 1 $ snd courseNames)),
+            title  = Nothing,
             prereqString = Nothing,
             f = Nothing,
             s = Nothing,
             y = Nothing,
-            name = T.pack $ fst courseNames,
+            name = T.pack "",
             exclusions = Nothing,
             manualTutorialEnrol = Nothing ,
             distribution = Nothing,
             prereqs = Nothing
         }
-    where
-        --remove [12l/24t] or similar from tag containing course code and name
-        removeLectureSection (Just (TagText s)) = takeWhile (/= '[') s
-
-        removeTitleGarbage s = replace "\160\160\160\160" " " s
-
-        cleanTag (TagText s) = TagText (replace "\r\n                   " " " s)
+    in snd (parsePrerequisite (parseDescription (parseTitle (tags, course))))
 
+-}
 main :: IO ()
 main = do
     rsp <- simpleHTTP (getRequest fasCalendarURL)
               -- fetch document and return it (as a 'String'.)
     body <- getResponseBody rsp
     let depts = doStuff body
-    --getCalendar $ filter (== "crs_csc.htm") depts
-    mapM_ getCalendar depts
-
-
-{-
-doStuff -> getDeptList -> map getCalendar 
--}
+    getCalendar $ filter (== "crs_csc.htm") depts
```

Comment:
I got a compilation error here. I generally don't like seeing compile errors pushed, since it's usually very easy to detect.